cockroachcloud/cloud-org-sso.md

```diff
@@ -30,31 +30,50 @@ Cloud Organization SSO allows you to customize your SSO configuration to meet yo
+{{ site.data.products.db }} users are identified by their email address. To reduce the risk of duplicated users, ensure that users have unique email addresses before you enable SSO autoprovisioning for an authentication method. If duplicate users result from enabling SSO autoprovisioning, you must delete them manually. Refer to [Manage Team Members](/docs/cockroachcloud/console-access-management.html#manage-team-members).
 {{site.data.alerts.end}}
 
-Cockroach Labs recommends that you enable autoprovisioning on only a single SSO method at a time, and that you migrate your users gradually. Most organizations aim to manage users in a single centralized IdP. It may be necessary to temporarily enable autoprovisioning to migrate a group of users from your centralized IdP who have yet not been onboarded to your Cloud organization.
+Cockroach Labs recommends that you enable SSO autoprovisioning on only a single SSO method at a time, and that you migrate your users gradually. Most organizations aim to manage users in a single centralized IdP. It may be necessary to temporarily enable SSO autoprovisioning to migrate a group of users from your centralized IdP who have yet not been onboarded to your {{ site.data.products.db }} organization.
 
-If you [require SSO authentication](configure-cloud-org-sso.html#require-sso), then when you deprovision a member from your IdP, they can no longer access your {{ site.data.products.db }} organization and there is no need to manually remove them from your {{ site.data.products.db }} organization.
+If you [require SSO authentication](configure-cloud-org-sso.html#require-sso) and disable password authentication, then when you deprovision a member from your IdP, they can no longer access your {{ site.data.products.db }} organization and there is no need to manually remove them from your {{ site.data.products.db }} organization, but some organizations choose to do so.
 
-{{site.data.alerts.callout_success}}
-If your organization includes members whose identity you don't manage, such as partners or consultants, you can leave password authentication enabled for those members, while instructing your internal users to sign in using SSO. Members who sign in using a password must be removed from your {{ site.data.products.db }} organization manually.
+If you require automatic deprovisioning or other automated user management features, refer to [SCIM autoprovisioning](#scim-autoprovisioning). Cockroach Labs does not recommend enabling both SSO autoprovisioning and SCIM autoprovisioning for the same authentication method.
```

Comment:
I don't think we'll need to recommend against enabling auto-provisioning and using SCIM. SCIM will not automatically remove users that are not in Okta